src/prefix-color-selector.ts

```diff
@@ -0,0 +1,77 @@
+import chalk from 'chalk';
+
+export class PrefixColorSelector {
+    lastColor: string;
+    autoColors: string[];
+
+    get ACCEPTABLE_CONSOLE_COLORS() {
+        // Colors picked randomly, can be amended if required
+        return (
+            [
+                chalk.cyan,
+                chalk.yellow,
+                chalk.magenta,
+                chalk.grey,
+                chalk.bgBlueBright,
+                chalk.bgMagenta,
+                chalk.magentaBright,
+                chalk.bgBlack,
+                chalk.bgWhite,
+                chalk.bgCyan,
+                chalk.bgGreen,
+                chalk.bgYellow,
+                chalk.bgRed,
+                chalk.bgGreenBright,
+                chalk.bgGrey,
+                chalk.blueBright,
+            ]
+                // Filter out duplicates
```

Comment:
is this necessary because chalk will return same colour depending on terminal support?

Reply:
It's been a while but I feel like this was probably to guard against human error, I might have been imaging the list of colours growing over time so it's difficult to manually see duplicates

Reply:
Makes sense, IMO 👍 We could use a [`Set`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) for this.

Reply:
updated